models/action.js

```diff
@@ -15,6 +15,7 @@ const ActionSchema = new Schema({
   user_id: String,
   field: String, // Used when an action references a particular field of an object. (e.g. a flag on a username or bio)
   detail: String, // Describes the reason for an action (e.g. 'Username is offensive')
+  description: String, // Describes the reason for the action in more detail (e.g. A description of why the comment is offensive.)
```

Comment:
Additionally, `details` tells me nothing about what that field contains or who made it. `comment` is clear as it's the "comment" related to the flagging action.